### PR TITLE
feat: Show correct currency in layout

### LINKED ISSUE #340: Display proper currency in app
Now the JSONAPI support is complete and the proper currency should be shown in event sale and attendee ticket detail

### PR DESCRIPTION
**Pull request by iamareebjamal**
- Add currency to ContextManager
- Add CurrencyUtils to correctly load currency symbol
- Change bus type to behaviour subject
- Delegate event loading to bus subscription
- Update layouts

Fixes #340

screenshot-1500861706956
screenshot-1500861719541

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@mejariamol Please review last commit

File: app/src/main/java/org/fossasia/openevent/app/main/MainPresenter.java [hunk 1]
```diff
@@ -52,15 +54,18 @@ public void start() {
             eventRepository
                 .getEvent(storedEventId, false)
                 .compose(dispose(getDisposable()))
-                .compose(erroneousResult(getView()))
```
Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/main/MainPresenter.java]: this is responsible for showing event name and date in navigation header.

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/main/MainPresenter.java]: See the subscribe of this rx stream.

Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/main/MainPresenter.java]: It doesn't call view's `showResult`?

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/main/MainPresenter.java]: It does -> `.compose(erroneousResult(getView()))`

Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/main/MainPresenter.java]: Yeah, correct. It was already in there. My bad :smile:

**Comment by iamareebjamal** [comment]
@niranjan94 Please merge

**Pull request merged**